Contest_API.md

```diff
@@ -371,6 +371,15 @@ attributes that are:
     In the latter case that can either be because it was a left out
     `null` value or because it was not implemented.
 
+### Filtering
+
+Endpoints that return a JSON array must allow filtering on any
+attribute with type ID (except the `id` field) by passing it as a
```

Comment:
What about attributes that are an array of IDs, like `group_ids` in the teams endpoint and `team_ids` in awards?
I'd argue to start simple and not support these yet.

Reply:
Thats what I only mention ID. I could explicitly state that filtering on array of ID does not have to be supported

Reply:
Added a note about array of ID